src/EventProducerService.cpp

```diff
@@ -90,22 +118,29 @@ void EventProducerService::sendEvent(bool state, byte index)
   }
 }
 
-void EventProducerService::sendEvent(bool state, byte index, byte data1)
+void EventProducerService::sendEvent(bool state, byte evValue, byte data1)
 {
+  byte index = module_config->findExistingEventByEv(1, evValue);
   if (index < module_config->EE_MAX_EVENTS)
   {
     byte opCode;
     byte nn_en[4];
     module_config->readEvent(index, nn_en);
+    //DEBUG_SERIAL << F("eps>index = ") << index << F(" , Node Number = 0x") << _HEX(nn_en[0]) << _HEX(nn_en[1]) << endl;
     if ((nn_en[0] == 0) && (nn_en[1] == 0))
     {
-      opCode = (state ? OPC_ASON1 : OPC_ASOF1);
+      opCode = (state ? OPC_ASON : OPC_ASOF);
```

Comment:
Shouldn't these opcodes be OPC_ASON1 and OPC_ASOF1 here? We are sending one byte of data.
I guess this is a copy paste error.
See also similar lines below.

Reply:
Well caught!  Now corrected through 1, 2 & 3.